drivers/storage/class/disk/disk.inf

```diff
@@ -212,3 +212,20 @@ compaq_array.DeviceDesc = "Compaq SCSI дисковый массив"
 mylex_array.DeviceDesc = "Mylex RAID дисковое устройство"
 
 VHD_Generic_FriendlyName = "Microsoft виртуальный диск"
+
+[Strings.0a]
+MSFT="Microsoft"
+DiskClassName="Unidades de discos"
+GenManufacturer="(Unidades de disco estándar)"
+Compaq="Compaq"
+Mylex="Mylex"
```

Comment:
```suggestion
```

Reply:
Same as the past one. Avoid dupe vendors. :)